public/audioViewApi.html

```diff
@@ -102,7 +102,8 @@ <h4><small id="text_name"></small></h4>
           source: 'custom_embed',
           action,
           data,
-          key: parseQueryString()['k']
+          key: parseQueryString()['k'],
+          isAnsRequired: true,
```

Comment:
I would prefer we only send this for the ready message

Reply:
I remember we discussed that we should have `isAnsRequired` option in the root of object instead under second level such as `data:{}` or `config: {}`
Also, since currently Composer will only send Story ANS to iframe when `action = 'ready` in addition of `isAnsRequired = true`, I believe that this part is being covered properly.

Reply:
This `sendMessage` is a generic function, I want u to only send this property if the the action is ready.
The better example is to make this change for the search since that should be the real use case for our clients

Reply:
Just to clarify, that we should remove `isAnsRequired` flag entirely from all `*Api.html` files, but in `*SearchApi.html` to have `isAnsRequired = true` to send out data in slightly different way?

I've quickly modify the lines for when action is ready, how does it look?
```javascript
        // Emulate wait time
        setTimeout(function() {
          window.parent.postMessage(
            JSON.stringify({
              source: 'custom_embed',
              action: 'ready',
              data: { height: document.documentElement.scrollHeight },
              key: parseQueryString()['k'],
              isAnsRequired: false,
            }),
            '*'
          )
        }, Number.parseInt(parameters.wait) + 100)
```

Reply:
here https://github.com/washingtonpost/arc-custom-embed/blob/3b5d4f09745df361566c0d740d22fb0c2f7f3d81/public/searchApi.html#L212-L214 right? Yes, that works.

Or if u wanna reuse sendMessage, u can change here https://github.com/washingtonpost/arc-custom-embed/blob/3b5d4f09745df361566c0d740d22fb0c2f7f3d81/public/searchApi.html#L120-L130
```
const messagePayload = {
  source: 'custom_embed',
  action,
  data,
   key: parseQueryString()['k']
}
if (action === 'ready') {
  messagePayload.isAnsRequired = true
}
```

Reply:
Are u still working on this part?

Reply:
Yes, but with question.

Are you suggesting to make changes on the audioViewApi/audioSearchApi/audioEditApi.js files with `isAnsRequired` flag included in the code? Or are we just documenting this suggestion?

Reply:
to update the audioSearchApi.js so clients can see an example